src/System.Diagnostics.DiagnosticSource/tests/DiagnosticSourceEventSourceBridgeTests.cs

```diff
@@ -0,0 +1,557 @@
+﻿using System;
+using System.Collections.Generic;
+using Xunit;
+using System.Threading.Tasks;
+using System.Diagnostics.Tracing;
+using System.Text;
+
+namespace System.Diagnostics.Tests
+{
+    public class DiagnosticSourceEventSourceBridgeTests
+    {
+        /// <summary>
+        /// Tests the basic functionality of turning on specific EventSources and specifying 
+        /// the events you want.
+        /// </summary>
+        [Fact]
+        public void TestSpecificEvents()
+        {
+            using (var eventSourceListener = new TestDiagnosticSourceEventListener())
+            using (var diagnosticSourceListener = new DiagnosticListener("BridgeTestSource1"))
+            {
+                Assert.Equal(0, eventSourceListener.EventCount);
+
+                // Turn on events with both implicit and explicit types 
+                eventSourceListener.Enable(
+                    "BridgeTestSource1/TestEvent1:cls_Point_X=cls.Point.X;cls_Point_Y=cls.Point.Y\r\n" +
```

Comment:
Can we use Environment.NewLine rather than ```"\r\n"``` in the test, to make sure this actually works correctly with the newlines on unix?